assets/js/atomic/blocks/product-elements/add-to-cart-form/editor.scss

```diff
@@ -1,30 +1,4 @@
-.wc-block-editor-add-to-cart-form {
-	display: flex;
-	flex-direction: column;
-	row-gap: $default-block-margin;
-}
-
-input.wc-block-editor-add-to-cart-form__quantity[type="number"] {
-	max-width: 50px;
-	min-height: 23px;
-	float: left;
-	padding: 6px 6px 6px 12px;
-	margin-right: 10px;
-	font-size: 13px;
-	height: inherit;
-}
-
-input[type="number"]::-webkit-inner-spin-button {
-	opacity: 1;
-}
-
-button.components-button.wc-block-add-to-cart-form__button {
-	float: left;
-	padding: 20px 30px;
-	border-radius: 0;
-}
-
-.wc-block-editor-container {
```

Comment:
Since some of this code was added to fix: https://github.com/woocommerce/woocommerce-blocks/pull/8582, I'd add a testing step to make sure the button styling is correct both with Gutenberg plugin disabled and enabled.